oe_theme.theme

```diff
@@ -602,3 +613,22 @@ function oe_theme_preprocess_paragraph__oe_content_row__variant_inpage(&$variabl
 
   $variables['links'] = $links;
 }
+
+/**
+ * Provide hook theme suggestions for patterns.
+ *
+ * @see ui_patterns_theme_suggestions_alter()
+ */
+function oe_theme_ui_patterns_suggestions_alter(array &$suggestions, array $variables, PatternContext $context) {
+  if ($context->isOfType('preview')) {
+    // Loop through each already existing suggestions.
+    array_walk(
+      $suggestions,
+      function (&$suggestion) {
+        $suggestion .= '__preview';
+      }
+    );
+
+    $suggestions[] = $variables['theme_hook_original'] . '__preview';
```

Comment:
@drupol as discussed I've added this upstream in UI Patterns (latest dev-master) so you can remove this hook implementation.

Reply:
Done bruh.